repo/packages/C/calico/2/config.json

```diff
@@ -0,0 +1,245 @@
+{
+  "properties": {
+    "Framework Settings": {
+      "description": "Configuration options for Calico's DC/OS Framework.",
+      "properties": {
+        "framework-name": {
+          "type": "string",
+          "description": "The name of the framework.",
+          "default": "calico-install-framework"
+        },
+        "master": {
+          "default": "zk://master.mesos:2181/mesos",
```

Comment:
We do round-robin on the backend depending on the number of zks someone is running (in a single master cluster, they all resolve to the same address). We're doing this because of issues in zk drivers around network partitions and dynamic masters.